### ISSUE TITLE
The link to the producers/consumers document in the howto index does not include the words "producers" or "consumers"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4227]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#4227|
|Type|enhancement|
|Created|2010-01-19 17:31:38Z|

The title of the document is:

"Producers and Consumers: Efficient High-Volume Streaming"

The link in the index is:

"Efficient High-Volume Streaming"

The link should be updated.

Attachments:

* [producer-link.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ee9/ee9e0798bad3a3cd3e36269fdad06f969b9141a0/72da68f1bbe489a4494c5cbeb408743b22a10734.patch) (580 bytes) - added by thijs on 2011-05-13 16:18:21Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4227 4227
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1263922298000000 1263922298000000
changetime__1305495387000000 1305495387000000
version__None None
owner__thijs thijs
cc__thijs cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|ivank set owner to @thijstriemstra|
|-|-|

1. This needs an empty news file `twisted/topfiles/4227.misc`

Otherwise, looks for merging.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [31905]) Apply producer-link.patch: Update the title of the producers howto in the index to reflect the document name.

Author: thijs
Reviewer: ivank
Fixes: [#4227](https://github.com/twisted/twisted/issues/4227)
